olp-cpp-sdk-dataservice-read/src/generated/api/BlobApi.cpp

```diff
@@ -29,14 +29,14 @@
 namespace olp {
 namespace dataservice {
 namespace read {
-using namespace olp::client;
 
-BlobApi::DataResponse BlobApi::GetBlob(const client::OlpClient& client,
-                                       const std::string& layer_id,
-                                       const std::string& data_handle,
-                                       boost::optional<std::string> billing_tag,
-                                       boost::optional<std::string> range,
-                                       const client::CancellationContext& context) {
+namespace client = olp::client;
+
+BlobApi::DataResponse BlobApi::GetBlob(
+    const client::OlpClient& client, const std::string& layer_id,
+    const std::string& data_handle, boost::optional<std::string> billing_tag,
+    boost::optional<std::string> range,
+    const client::CancellationContext& context) {
```

Comment:
I don't understand this back and forth formatting.

Reply:
clang-format 8.0.0 was used previously